shop/.gitignore

```diff
@@ -0,0 +1,46 @@
+# See http://help.github.com/ignore-files/ for more about ignoring files.
+
+# compiled output
```

Comment:
I would clear this file completely and only add the things you need now.

/node_modules
/dist